core/src/dht/dht_reducers.rs

```diff
@@ -39,8 +46,11 @@ pub fn reduce(
 /// Maps incoming action to the correct reducer
 fn resolve_reducer(action_wrapper: &ActionWrapper) -> Option<DhtReducer> {
     match action_wrapper.action() {
+        Action::Commit(_) => Some(reduce_add_crud_meta),
```

Comment:
This should not happen on `Commit`, but on `Hold`. The DHT should not listen to `Commit`. That was short cut that doesn't make sense when we have networking.

Reply:
I don't think that works because `Hold` can be called for anything: create, modify delete.
I need to do a specific thing only when its a 'create' and `Commit` seemed to do just that.
Generally I see a confusion between 'Commit' and 'Create' which needs to be cleared out.